styles/hydrogen.less

```diff
@@ -96,6 +96,8 @@
     .bubble-output-container {
         color: lighten(@syntax-text-color, 10%);
         background-color: @bubble-background;
+        // in order to don't accept the shadow from action-panel
```

Comment:
Oh sorry my english isn't that great. I meant: `in order to be above the shadow of action panel`